### PR TITLE
network: drop unnecessary nexthops

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Comment by keszybz** [comment]
I assume this will be rebased after the other PRs are merged.

**Comment by poettering** [comment]
i figure you are in the process of splitting this up in smaller PRs, and we should only review those?

**Comment by yuwata** [comment]
@poettering 
> i figure you are in the process of splitting this up in smaller PRs, and we should only review those?

Yes. But it is also OK to review and merge this.

**Comment by yuwata** [comment]
This is the final part. Rebased. PTAL.

File: test/test-network/conf/25-nexthop-nothing.network [hunk 1]
```diff
@@ -0,0 +1,74 @@
+[Match]
```
Comment (keszybz) [inline test/test-network/conf/25-nexthop-nothing.network:1]: Is this chunk supposed to be in this commit? This file is new and doesn't seem to be referenced anywhere.

**Review by keszybz — COMMENTED** [review]
LGTM, except for the one commit-health thing.

**Comment by yuwata** [comment]
@keszybz Thank you for your review. The .network file for testing is moved to the right commit. Upgrading the green label.

**Comment by keszybz** [comment]
Let's merge this after -rc2.

**Comment by keszybz** [comment]
bionic-amd64 timed out in "upstream" tests, and bionic-i386 failed in TEST-50-DISSECT. Both look unrelated.

**Pull request merged**